layout/password.liquid

```diff
@@ -65,6 +65,15 @@
 
         --page-width: {{ settings.page_width | divided_by: 10 }}rem;
         --page-width-margin: {% if settings.page_width == '1600' %}2{% else %}0{% endif %}rem;
+
+        --text-boxes-padding: {{ settings.text_boxes_padding }}px;
```

Comment:
I personally think anything padding/margin related should be in relative units for accessibility and better responsiveness. This will allow the page to scale appropriately for users who use zoom, or if the font size changes.

Reply:
We will figure this out with UX and then make a broad decision. Great call-out Martina.